From 6c4b7aba10067eb0b03185d932f4a2ce0fb4d593 Mon Sep 17 00:00:00 2001
From: Charles Lamb <clamb@cloudera.com>
Date: Mon, 14 Apr 2014 14:50:38 -0400
Subject: [PATCH 1578/1612] HDFS-5276. FileSystem.Statistics should use thread-local counters to avoid multi-threaded performance issues on read/write.  (Colin Patrick McCabe)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1533670 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit fa07c771b067e330f4af82f3c15ed0460ef84456)

Conflicts:

	hadoop-common-project/hadoop-common/CHANGES.txt
---
 .../main/java/org/apache/hadoop/fs/FileSystem.java |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java
index 6ee236b..866b442 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java
@@ -2811,6 +2811,7 @@ public abstract class FileSystem extends Configured implements Closeable {
       });
     }
 
+
     @Override
     public String toString() {
       return visitAll(new StatisticsAggregator<String>() {
-- 
1.7.0.4

